### DISCUSSION TITLE
Use Sonoff Basic to toggle an External relay.

### DISCUSSION THREAD

**Discussion by Daddel15** [arendst/Tasmota#14200] [Q&A] [answered]
Hi,
first things first. Sry for my bad english.
I have a little problem. I need to toggle a relay with a sonoff basic. Therefore I have to change the behavior of the web toggle to work as a button instead of a switch.
Up to this point I managed to soder a small Button to the gpio03 header an toggle the relay by that but I have no clue how to change the behavior of the "virtuale" switch in the web interface (I need this since I want to use the button via MQTT/homeassistant). 
help is much appreciated

**Comment by barbudor** [comment, marked as answer]
What do you mean by "web toggle to work as a button instead of a switch"?
By default it is already working as a Button, toggling the relay each time you click on it.
